cmd/segment-reaper/bitmask_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBitmask(t *testing.T) {
+	t.Run("Set", func(t *testing.T) {
+		t.Run("ok", func(t *testing.T) {
+			var (
+				expectedIdx = rand.Intn(64)
+				mask        bitmask
+			)
+
+			err := mask.Set(expectedIdx)
+			require.NoError(t, err)
+		})
+
+		t.Run("error: negative index", func(t *testing.T) {
+			var (
+				invalidIdx = -(rand.Intn(math.MaxInt32-1) + 1)
+				mask       bitmask
+			)
+
+			err := mask.Set(invalidIdx)
+			assert.Error(t, err)
+			assert.True(t, errorBitmaskInvalidIdx.Has(err), "errorBitmaskInvalidIdx class")
+		})
+
+		t.Run("error: index > 63", func(t *testing.T) {
+			var (
+				invalidIdx = rand.Intn(math.MaxInt16) + 64
+				mask       bitmask
+			)
+
+			err := mask.Set(invalidIdx)
+			assert.Error(t, err)
+			assert.True(t, errorBitmaskInvalidIdx.Has(err), "errorBitmaskInvalidIdx class")
+		})
+	})
+
+	t.Run("Has", func(t *testing.T) {
+		t.Run("ok", func(t *testing.T) {
+			var (
+				expectedIdx = rand.Intn(64)
+				mask        bitmask
+			)
+
+			has, err := mask.Has(expectedIdx)
+			require.NoError(t, err)
+			assert.False(t, has)
+		})
+
+		t.Run("error: negative index", func(t *testing.T) {
+			var (
+				invalidIdx = -(rand.Intn(math.MaxInt32-1) + 1)
+				mask       bitmask
+			)
+
+			_, err := mask.Has(invalidIdx)
+			assert.Error(t, err)
+			assert.True(t, errorBitmaskInvalidIdx.Has(err), "errorBitmaskInvalidIdx class")
+		})
+
+		t.Run("error: index > 63", func(t *testing.T) {
+			var (
+				invalidIdx = rand.Intn(math.MaxInt16) + 64
+				mask       bitmask
+			)
+
+			_, err := mask.Has(invalidIdx)
+			assert.Error(t, err)
+			assert.True(t, errorBitmaskInvalidIdx.Has(err), "errorBitmaskInvalidIdx class")
+		})
+	})
+
+	t.Run("Set and Has", func(t *testing.T) {
```

Comment:
May I suggest a few more test cases:
- set the same bit twice - check that it is still set
- set several bits - check that all of them are still set at the end.